src/components/button/_button.scss

```diff
@@ -202,12 +202,12 @@
 
   .#{$prefix}--btn--secondary {
     @include button-theme--x(
-      $ibm-colors__gray-80,
-      $ibm-colors__gray-80,
+      $ibm-color__gray-80,
```

Comment:
I have seen that many consumers of `carbon-components` have made changes to cope with our earlier color change, and reverting it back now causes another breaking change. Quick test of adopting code from this PR to IBM Cloud header shows a build failure. Any way to avoid that...?

Reply:
Any point-of-contact for me to reach out to? I had assumed ibm color variables were internal, and the color variables we expose were available at `_colors.scss`.

Reply:
should those consumers just be able to just use `carbon-elements` to build their own color maps instead then? although if these values were supposed to be internal-only then they take a risk by using them directly right?

Reply:
@emyarod good point. They totally could use elements if they want, the official color names are going to be: `$carbon--` namespaced. We are doing `ibm-color` for interop with old IDL code. There will also be aliases for colors so that `carbon--blue-50` can also be used as `blue-50`.

Reference: https://github.com/IBM/carbon-elements/tree/master/packages/colors#sass

---

I went ahead and added back in ibm-colors so that we can proceed here 👍

Reply:
> I went ahead and added back in ibm-colors so that we can proceed here 👍

I may be misunderstanding but the diff still contains both `ibm--colors` as well as `ibm--color`, is that correct?

Reply:
I assume that it would be better to name what we want now rather than perhaps having to deal with it later as v10 is a major breaking change anyway. As Andrew said they did take a risk using them directly. If we are wanting to push for use of use of carbon elements and our official name tokens then I don't see the issue with it being color vs colors.

Reply:
@emyarod yeah, it contains both for now so we can move forward without breaking.

@aledavila yeah, I definitely agree. If reaching out can help them though, I don't mind doing it to reduce frustration. Our public sass API is not documented so it's on us to define it for v10 so this doesn't happen in the future.

Reply:
@joshblack @emyarod @aledavila Given teams need IBM color refs, I see it's natural for historical Carbon consumers using our IBM color variables. We need to consider any variables as APIs unless we explicitly tell consumers. Given this PR has been merged anyway, let's see how teams respond (they just made application-side change last time as it was simple search/replace change I guess). @joshblack the contact is @flannanl